src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT.java

```diff
@@ -0,0 +1,119 @@
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.getLabel;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.handleClassLogs;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.initClient;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.initCloud;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.initCredentials;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.instanceConfiguration;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.logs;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.api.services.compute.model.Instance;
+import com.google.jenkins.plugins.computeengine.ComputeEngineCloud;
+import com.google.jenkins.plugins.computeengine.client.ComputeClient;
+import hudson.model.labels.LabelAtom;
+import hudson.slaves.NodeProvisioner.PlannedNode;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+import java.util.logging.StreamHandler;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT {
+  private static Logger log =
+      Logger.getLogger(ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT.class.getName());
+
+  private static final String MULTIPLE_NUM_EXECUTORS = "2";
+  private static final String EXPECTED_LOG_MESSAGE = "for excess workload of 2 units of label";
+
+  @ClassRule public static Timeout timeout = new Timeout(5, TimeUnit.MINUTES);
+  @ClassRule public static JenkinsRule jenkinsRule = new JenkinsRule();
+
+  private static ByteArrayOutputStream logOutput = new ByteArrayOutputStream();
+  private static StreamHandler streamHandler = new StreamHandler(logOutput, new SimpleFormatter());
+  private static ComputeClient client;
+  private static Map<String, String> label =
+      getLabel(ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT.class);
+  private static Instance instance;
+
+  @BeforeClass
+  public static void init() throws Exception {
+    log.info("init");
+    initCredentials(jenkinsRule);
+    ComputeEngineCloud cloud = initCloud(jenkinsRule);
+    handleClassLogs(streamHandler, ComputeEngineCloud.class.getName());
+    client = initClient(jenkinsRule, label, log);
+    handleClassLogs(streamHandler, ComputeClient.class.getName());
+
+    cloud.addConfiguration(
+        instanceConfiguration(
+            DEB_JAVA_STARTUP_SCRIPT,
+            MULTIPLE_NUM_EXECUTORS,
+            LABEL,
+            label,
+            false,
+            false,
+            NULL_TEMPLATE));
+    // Add a new node
```

Comment:
Comments which simply describe what the code is doing don't add much value. The code should be readable enough on its own to make that clear, if not it should be re-factored. Rather, reserve in-line comments like these for rationale and/or points of clarification which otherwise wouldn't be clear from reading the code. See: TotT #563.

Reply:
Disclaimer, I'm often guilty of doing this and am working on this too :)

Reply:
This feedback applies throughout this PR.

Reply:
Thanks, there are many comments like this throughout that are either copy-pasted from the original test methods or added by me for consistency with the others, so I'll go through and clean these up.